build-system/git.js

```diff
@@ -60,17 +60,23 @@ exports.gitDiffStatMaster = function() {
 
 /**
  * Returns a detailed log of commits included in a PR check, starting with (and
- * including) the branch point off of master.
+ * including) the branch point off of master. Limited to at most 100 commits to
+ * keep the output sane.
  *
  * @return {string}
  */
 exports.gitDiffCommitLog = function() {
+  const maxCount = 100;
   const branchPoint = process.env.TRAVIS ?
     exports.gitPrBranchPoint() : exports.gitBranchPointFromMaster();
-  return getStdout(`git -c color.ui=always log --graph --pretty=format:\
-"%C(red)%h%C(reset) %C(bold cyan)%an%C(reset) -%C(yellow)%d%C(reset) \
-%C(reset)%s%C(reset) %C(green)(%cr)%C(reset)" \
---abbrev-commit ${branchPoint}^...HEAD`).trim();
+  let commitLog = getStdout(`git -c color.ui=always log --graph \
+--pretty=format:"%C(red)%h%C(reset) %C(bold cyan)%an%C(reset) \
+-%C(yellow)%d%C(reset) %C(reset)%s%C(reset) %C(green)(%cr)%C(reset)" \
+--abbrev-commit ${branchPoint}^...HEAD --max-count=${maxCount}`).trim();
+  if (commitLog.split('\n').length >= maxCount) {
```

Comment:
Do you count the boilerplate/prefix lines here?

Reply:
This didn't work, so I removed it.